src/applications/caregivers/actions/fetchFacilities.js

```diff
@@ -46,15 +46,15 @@ export const fetchFacilities = async ({
         };
       });
     })
-    .catch(({ error }) => {
+    .catch(error => {
       Sentry.withScope(scope => {
         scope.setExtra('error', error);
         Sentry.captureMessage(content['error--facilities-fetch']);
       });
 
       return {
         type: 'SEARCH_FAILED',
-        errorMessage: error,
+        errorMessage: 'There was an error fetching the health care facilities.',
```

Comment:
We still need confirmation on what the wording for this generic error message should be when the facilities request returns an error.